timebot/timebot.go

```diff
@@ -3,10 +3,24 @@
 package timebot
 
 import (
+	"fmt"
 	"time"
+	"strings"
 )
 
 // ParseTime takes a string and returns time.Time in time.UTC
 func ParseTime(text string) (time.Time, bool) {
-	return time.Time{}, true
+	const longForm = "2006-01-02 15:04 -0700"
+
+	text = strings.Replace(text, "PST", "-0800", 1)
+	text = strings.Replace(text, "KST", "+0900", 1)
+	
+	isParsed := true
+	t, err := time.Parse(longForm, text)
+	if err != nil {
+		fmt.Println(err)
+		isParsed = false
+	}
+	t = t.UTC()
```

Comment:
이 부분도 마이너해서 그냥 변경안하셔도 됩니다 논의를 위해 :smile: 


 한줄 추가 되지 않는게 좀더 idiomatic 해보입니다

```suggestion
	return t.UTC(), false
```

Reply:
요거랑 저 위에꺼 변경해서 다시 풀리퀘하겠습니다. 
무엇보다 제가 맘에 듭니다. :-)